yabause/src/sh2int.c

```diff
@@ -252,8 +252,10 @@ static void FASTCALL SH2delay(SH2_struct * sh, u32 addr)
 #endif
 
    // Execute it
-   sh->regs.PC -= 2;
+   u32 tmp = sh->regs.PC;
```

Comment:
This is the fix of #268 .
https://github.com/devmiyax/yabause/commit/d8122116942c6326b7d508270fd6818fb014ab92